src/containers/PermissionsApplication.jsx

```diff
@@ -29,7 +29,21 @@ const PermissionsApplication = ({ match }) => {
           {t('Permissions.failedRequest')}
         </Typography>
       ) : (
-        <PageTitle>{appName}</PageTitle>
+        <div>
+          <PageTitle>{appName.toUpperCase()}</PageTitle>
```

Comment:
nit: This is not really the appName. This is the `appSlug`. 

Exemple: https://github.com/cozy/cozy-drive/blob/master/src/drive/targets/manifest.webapp#L2-L4

If you really want, you can display the real appName: https://github.com/cozy/cozy-drive/blob/master/src/drive/targets/manifest.webapp#L2-L4 https://github.com/cozy/cozy-notes/blob/master/manifest.webapp#L2-L4 . You'll not need this toUpperCase() method.